### PR TITLE
Update README.md to add Gradle to the install section

### PR DESCRIPTION
**Pull request by QBourdette**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +94,10 @@ Add the repo to your project inside your `pom.xml` file
 </dependency>
 ```
 
+##### Gradle
+Add the repo to your project by adding `mavenLocal()` to `repositories` and `implementation 'com.sipios:spring-search:0.0.4-SNAPSHOT'` to  `dependencies` in your `build.gradle` file.
```
Comment (NH0) [inline README.md]: 0.2.0 instead of 0.0.4-SNAPSHOT

Comment (NH0) [inline README.md]: 0.2.0 instead of 0.0.4-SNAPSHOT

File: README.md [hunk 2]
```diff
@@ -93,6 +94,10 @@ Add the repo to your project inside your `pom.xml` file
...
+
+
```
Comment (NH0) [inline README.md]: two spaces

File: README.md (hunk 1 above)
Comment (QBourdette) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (QBourdette) [reply README.md]: fixed

**Pull request merged**